README.md

```diff
@@ -13,7 +13,7 @@ Several things will happen if one elects to continue:
     git commit --message 'Version 0.6.1'
     git tag --annotate 'v0.6.1' --message 'Version 0.6.1'
     git push origin 'refs/heads/master' 'refs/tags/v0.6.1'
-    npm publish
+    npm publish # Only for non-private packages.
```

Comment:
Not sure if we need a more elegant way to document this feature. It's kind-of intuitive behavior.